status/ss13status.py

```diff
@@ -3,7 +3,7 @@
 import struct
 import socket
 import urllib.parse
-import html.parser as htmlparser
+import html as htmlparser
```

Comment:
I couldn't find any immediate sources stating that html.parser is being scrapped. The current 3.11 dev docs still reference it as being an active module. Can you please cite your source on that?